Samples/SampleCode/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: vtku3dexporter
+channels:
+  - clinicalgraphics
+  - conda-forge
```

Comment:
What packages do you specifically need from conda-forge? Do we need them at runtime too? If so, we should copy them to our own channel

Reply:
We also have this recipe on conda-forge, so once we solve this, we could/should update that one too. But yes, copying over the required packages is important